ddtrace/internal/telemetry/request.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
```

Comment:
Can we just include these methods in the `writer` module? They shouldn't be used elsewhere as far as I can tell.